### PR TITLE
Create migration to add name column to user table

### PR DESCRIPTION
**Pull request by kon-ham**
I ran `rails g migration` to add a `name` to the User table - this action was based off of the conversation between Gaelyn and I. 

Please let me know if this is inadequate or needs revision.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -224,6 +224,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-20
+  x86_64-darwin-21
```
Comment (gaelyn) [inline Gemfile.lock:227]: 👀  Does this serve a purpose?

**Review by gaelyn — APPROVED** [review]
Thanks Kon!

File: Gemfile.lock (hunk 1 above)
Comment (kon-ham) [reply Gemfile.lock:227]: Gaelyn, have you updated OSX? I think this came from my computer when I updated to the newest version. Perhaps when I ran `bundle install` it added support for darwin-21... I'm not sure exactly if it serves a purpose beyond being able to run on my 15" which has the latest version. Maybe we need it?

Comment (gaelyn) [reply Gemfile.lock:227]: Ah, that makes sense. I don't think I've updated. Ok, just curious! Thanks!

Comment (kon-ham) [reply Gemfile.lock:227]: So in terminal if you type `uname -r` you can check your version. Mine was set to darwin-21.1.0...what is yours?

**Pull request merged**